app/src/main/res/layout-xhdpi/activity_multimeter.xml

```diff
@@ -283,7 +283,10 @@
                 android:textStyle="bold"
                 app:layout_constraintCircle="@id/knobs"
                 app:layout_constraintCircleAngle="@string/multimeter_angle_12"
-                app:layout_constraintCircleRadius="@dimen/multimeter_knobcircle_radius_2_xl" />
+                app:layout_constraintCircleRadius="@dimen/multimeter_knobcircle_radius_2_xhdpi"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                tools:layout_editor_absoluteY="290dp"/>
```

Comment:
You might have missed to add this one to dimens.xml

Reply:
> Looks almost good

Sorry sir,actually it was added by mistake.It was not necessary.Removed it Sir.